ic-os/components/setupos-scripts/check-network.sh

```diff
@@ -44,7 +44,19 @@ function eval_command_with_retries() {
     done
 
     if [ -z "${result}" ]; then
-        log_and_halt_installation_on_error "1" "${error_message}"
+        local ip6_output=$(ip -6 addr show)
+        local ip6_route_output=$(ip -6 route show)
+        local dns_servers=$(cat /etc/resolv.conf | grep 'nameserver')
+
+        log_and_halt_installation_on_error "1" "${error_message}
+Output of 'ip -6 addr show':
```

Comment:
Will an NP know from looking at this what correct output should actually look like?

Reply:
If they have basic networking knowledge, I believe so. They should recognize their IPv6 address and which interface its associated with.